tables_list.csv

```diff
@@ -0,0 +1,96 @@
+Name,Description

```

Comment:
I thought we'll need the product name as well to filter out the relevant tables of the product. Confirming this since we still have the product selection logic in the code. I am assuming we have omitted that for now because we don't need to deal with multiple products.

Reply:
There are only 7-8 products. I would suggest we have different XMLs for products. 

Some tables may be part of multiple products. That is handled.